drivers/gles3/rasterizer_canvas_gles3.h

```diff
@@ -108,8 +108,8 @@ class RasterizerCanvasGLES3 : public RendererCanvasRender {
 		RID texture;
 		struct {
 			bool enabled = false;
-			float z_far;
-			float y_offset;
+			float z_far = 0.0;
+			float y_offset = 0.0;
```

Comment:
![image](https://github.com/godotengine/godot/assets/4701338/bd5f0148-ced2-428b-9011-f39bf8c8d4b9)

CC @clayjohn @BastiaanOlij

Reply:
I removed these changes from the PR for now.